modules/@apostrophecms/doc-type/index.js

```diff
@@ -1870,7 +1870,6 @@ module.exports = {
           after(results) {
             // In all cases we mark the docs with ._edit and ._publish if
             // the req is permitted to do those things
-            self.apos.permission.annotate(query.req, 'create', results);
```

Comment:
What about pages? For pages, `create` permission on the parent page is requires to create a page.

Reply:
Unless that is handled outside of this PR?

Reply:
I'm checking with Val.